DiamondLightSource/python-artemis
# Fix initialisation of OAV device and parameter loading

Fixes #422 

### To test:
1. Check OAVParameters is capable of loading in the correct beam centre, and microns to pixels for a given zoom.
2. Check PVs are being set to the correct values on run.
3. Check the zoom level can't be set to an unknown value (other than 1.0x, 2.5x, 5.0x, 7.5x, 10x).
